synapse/models/inet.py

```diff
@@ -1676,40 +1707,52 @@ def getModelDefs(self):
 
                     ('inet:service:base', {
                         'doc': 'Properties common to most forms within a service platform.',
+                        'template': {'service:base': 'node'},
                         'props': (
 
                             ('id', ('str', {'strip': True}), {
-                                'doc': 'A platform specific ID.'}),
+                                'doc': 'A platform specific ID which identifies the {service:base}.'}),
```

Comment:
It might be worth adding a test to make sure all of these (throughout the whole model) placeholders are getting replaced.

Reply:
Easier to write a test now than:
- get a customer report about the doc string being incorrect
- creating a ticket
- pointing ticket
- prioritizing ticket
- assigning ticket
- completing ticket
- releasing bugfix

Reply:
Added a warning. I don't think we can raise an exception due to the possibility of existing extended model elements which happen to overlap with the template syntax :+1: